apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -200,15 +202,13 @@ export default function sectionProgress(state=initialState, action) {
     };
   }
   if (action.type === SET_VALID_SCRIPTS) {
-    // If no scriptId is assigned, use the first valid script.
-    const defaultScriptId = state.scriptId || action.validScripts[0].id;
 
     let validScripts = action.validScripts;
     if (action.studentScriptIds && action.validCourses) {
-
+      // Include the id for the Accelerated Course so that there will always be // at least one validScript and we don't end up with an empty dropdown.
+      const idMap = {ACCELERATED_SCRIPT_ID: true};
```

Comment:
I'm not sure this will be popular from a product perspective. In the next PR, let's find a way to only add the accelerated course if the list would otherwise be empty.